src/ragQueries.ts

```diff
@@ -0,0 +1,42 @@
+import { GraphAnnotation } from "./agent/agent"
+import { OpenAIEmbeddings } from "@langchain/openai";
+import { Chroma } from "@langchain/community/vectorstores/chroma";
+
+const embeddings = new OpenAIEmbeddings({
+    model: "text-embedding-3-small",
+    apiKey: process.env.OPENAI_API_KEY || ""
+});
+
+const vectorStore = new Chroma(embeddings, {
+    collectionName: "uw-courses-test",
+    url: "http://localhost:8080", // Optional, will default to this value
```

Comment:
Lets move this into the .env file and use import `dotenv/config` to bring in these values